scijava/scijava-struct/src/main/java/org/scijava/struct/DefaultStructInstance.java

```diff
@@ -0,0 +1,43 @@
+
+package org.scijava.struct;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DefaultStructInstance<C> implements StructInstance<C> {
```

Comment:
We should definitely javadoc the struct stuff. Of course we need javadoc on all public API, but struct is super abstract and will be very helpful to have it documented for later.